test/groovy/NeoDeploymentTest.groovy

```diff
@@ -287,11 +231,11 @@ class NeoDeploymentTest extends BasePipelineTest {
                                 }
                                 
                                 return this
-                                """
+                                """ }
     }
 
     private noScriptPipeline(){
-        new File(pipeline) <<   """
+    { ->    """
```

Comment:
Indentation is different here than in the other 'Pipelines'
In general, IMO, there is too much white space for each line of these pipeline scripts.

Reply:
I agree, 4 white spaces should be enough.

Reply:
done